src/breadcrumbs/BreadcrumbLink.ts

```diff
@@ -1,6 +1,6 @@
 import { createComponent, createHook } from "reakit-system";
 
-import { LinkHTMLProps, LinkOptions, useLink } from "../link";
+import { LinkHTMLProps, LinkOptions, useLink } from "../Link";
```

Comment:
Linux systems does not care about case sensitive imports but windows does, so it was breaking the build.